cognite/client/data_classes/iam.py

```diff
@@ -140,7 +141,7 @@ def is_managed_in_cdf(self) -> bool:
         return self.members is not None
 
     @classmethod
-    def _load(cls, resource: dict, cognite_client: CogniteClient | None = None, allow_unknown: bool = False) -> Group:
+    def _load(cls, resource: dict, cognite_client: CogniteClient | None = None, allow_unknown: bool = True) -> Group:
```

Comment:
```suggestion
    def _load(cls, resource: dict, cognite_client: CogniteClient | None = None, allow_unknown: bool = False) -> Group:
```

Reply:
This is the fix, why do you want to revert that?